### PR TITLE
Add missing error code explanation for E0183

### PR DESCRIPTION
**Pull request by ewk**
This is part of the ongoing work for #61137

### CODE REVIEW

File: src/librustc_typeck/error_codes.rs [hunk 1]
```diff
@@ -1588,6 +1588,20 @@ fn bar(foo: Foo) -> u32 {
 ```
 "##,
 
+E0183: r##"
+Fn* traits (`unboxed closures`) are experimental.
+
+Functions must have exactly one (non self) argument, a tuple representing
```
Comment (GuillaumeGomez) [inline src/librustc_typeck/error_codes.rs]: Please add a failing code example.

File: src/librustc_typeck/error_codes.rs [hunk 2]
```diff
@@ -1588,6 +1588,20 @@ fn bar(foo: Foo) -> u32 {
...
+the argument list [1]. This feature can be enabled with:
+
+```
+#![feature(unboxed_closures)]
```
Comment (GuillaumeGomez) [inline src/librustc_typeck/error_codes.rs]: Please show a complete fixed code example.

**Review by GuillaumeGomez — CHANGES_REQUESTED** [review]
Please follow the same "template" the other long error explanations are using too please.

**Comment by chocol4te** [comment]
Ping from triage @ewk, waiting on changes.

**Comment by ProgrammaticNajel** [comment]
Ping from triage. @ewk any updates on this?

**Comment by Karrq** [comment]
I'll get it done tonight, if that's not a problem

**Comment by Dylan-DPC** [comment]
Closing this as @Karrq will work on it in a separate PR

**Pull request closed without merging**